packages/babel-parser/test/attachComment-false.js

```diff
@@ -0,0 +1,12 @@
+import path from "path";
+import { runFixtureTestsWithoutExactASTMatch } from "./helpers/runFixtureTests";
+import { parseExpression } from "../lib";
+import { fileURLToPath } from "url";
+
+runFixtureTestsWithoutExactASTMatch(
+  path.join(path.dirname(fileURLToPath(import.meta.url)), "expressions"),
+  (input, options = {}) => {
+    options.attachComment = false;
+    return parseExpression(input, options);
+  },
+);
```

Comment:
We run `parseExpression(#, { attachComment: false })` on `expressions` fixture as smoke tests. It is faster than running over the complete parser tests.